addie/calculate_gr/edit_sq_dialog.py

```diff
@@ -260,12 +243,11 @@ def do_set_scale_range(self):
             current_scale_factor = max_scale
 
         # TODO/ISSUE/NOW - clean up to multiple steps and check!
-        scale_factor_int = int(current_scale_factor/(max_scale - min_scale) *
-                               (self.ui.horizontalSlider_scale.maximum() - self.ui.horizontalSlider_scale.minimum()))
+        scale_factor_int = int(current_scale_factor/(max_scale - \
```

Comment:
This would be much clearer if you added two temporary variables for `max_scale - min_scale` and for `self.ui.horizontalSlider_scale.maximum() - self.ui.horizontalSlider_scale.minimum()`. It would also help explain if you mean to multiply or divide by the horizontal slider stuff.

Reply:
I'm not sure what Wenduo wanted to do here but I can add a few lines to make it clearer

Reply:
e44a7d3df4fbc0e8a3f8d08a2687896943ddfd38